crates/pallet-feeds/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+authors = ["Serge Kovbasiuk <serge@subspace.network>"]
+description = "Subspace node pallet for interacting with storage"
+edition = "2018"
+homepage = "https://subspace.network"
+license = "Apache-2.0"
+name = "pallet-feeds"
+readme = "README.md"
+repository = "https://github.com/subspace/subspace"
+version = "0.1.0"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = {package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"]}
+frame-support = {version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate", rev = "d73c37ecc16f4895c5f6ed5126d72af8cf87d9b3"}
```

Comment:
These dependencies are from older revision of `main`, they need to be updated to `fa635c968e831110b13f7ff997c4cd3f5e9a3798` (as of right now)